app/components/Footer.tsx

```diff
@@ -45,39 +32,10 @@ const socialMediaLinks = [
   },
 ];
 
-const aboutMenuItems = [
-  { text: "Mission & Purpose", href: "/about" },
-  { text: "News", href: "/news" },
-];
-
-const patientSupportMenuItems = [
-  { text: "Urgent Care", href: SAVE_MY_SPOT },
-  { text: "Specialty Treatment", href: "/specialties" },
-  { text: "Billing & Insurance", href: "/patient-care" },
-  { text: "MyChart", href: MYCHART_URL },
-];
-
-const healthProfessionalsMenuItems = [
-  { text: "LuskinOIC Physicians", href: "/providers" },
-  { text: "Refer a Patient", href: "/medical-professionals" },
-  { text: "Careers", href: CAREERS_URL },
-  { text: "Volunteer", href: "/ways-to-give" },
-  {
-    text: "Research",
-    href: "/research",
-  },
-];
-
-const getInvolvedMenuItems = [
-  { text: "Donate", href: "/ways-to-give" },
-  { text: "Events", href: "/events" },
-  { text: "Volunteer", href: "/ways-to-give" },
-];
-
 const styles = {
   footerContainer: "flex justify-evenly bg-[#0076AD] text-white py-8",
   leftContainer: "flex flex-col tracking-wide gap-2 pl-4 lg:pl-4",
-  rightContainer: "hidden lg:flex flex-row gap-x-20",
+  rightContainer: "hidden lg:grid grid-cols-2 md:grid-cols-2 gap-10",
```

Comment:
@cpualei, the PR description mentions there are no visual changes. What are this changes here for?

Reply:
Hi @jtorreggiani. That style change was meant to place the right side sections in columns.

This was how it looked before:
![Screenshot 2024-06-03 at 3 42 08 PM](https://github.com/LuskinOIC/website/assets/93879557/b822e21c-7d82-4b76-93e0-e306d3170b87)

And the style change I implemented placed them into two columns like how it looked prior to making the footer an editable Contentful component. Like below:
![Screenshot 2024-06-03 at 3 44 06 PM](https://github.com/LuskinOIC/website/assets/93879557/c3db84b2-5158-43cf-908c-6b1859a5057f)